Dockerfile

```diff
@@ -16,38 +16,30 @@ RUN apt-get update && \
     wget --no-cache -q -O /opt/altv/data/vehmods.bin https://cdn.altv.mp/data/${BRANCH}/data/vehmods.bin && \
     wget --no-cache -q -O /opt/altv/data/clothes.bin https://cdn.altv.mp/data/${BRANCH}/data/clothes.bin && \
     chmod +x /opt/altv/altv-server /root/entrypoint.sh && \
-    apt-get purge -y wget && \
-    apt autoremove -y && \
-    apt-get clean
-
-######
-# Install JS Module
-######
-RUN apt-get install -y wget jq && \
+    \
+    ######
+    # Install JS Module
+    ######
     mkdir -p /opt/altv/modules/js-module/ && \
     wget --no-cache -q -O /opt/altv/modules/js-module/libnode.so.${LIBNODE_VERSION} https://cdn.altv.mp/js-module/${BRANCH}/x64_linux/modules/js-module/libnode.so.${LIBNODE_VERSION} && \
     wget --no-cache -q -O /opt/altv/modules/js-module/libjs-module.so https://cdn.altv.mp/js-module/${BRANCH}/x64_linux/modules/js-module/libjs-module.so && \
     wget --no-cache -q -O /opt/altv/modules/js-module/libjs-bytecode-module.so https://cdn.altv.mp/js-bytecode-module/${BRANCH}/x64_linux/modules/libjs-bytecode-module.so && \
-    apt-get purge -y wget jq && \
-    apt autoremove -y && \
-    apt-get clean
-
-######
-# Install .NET 6 Module
-######
-RUN apt-get install -y wget gnupg && \
+    \
+    ######
+    # Install .NET 6 Module
+    ######
     # install dotnet runtime(s)
     wget https://packages.microsoft.com/config/debian/10/packages-microsoft-prod.deb -O packages-microsoft-prod.deb && \
     dpkg -i packages-microsoft-prod.deb && \
     rm -f packages-microsoft-prod.deb && \
     apt-get update && \
-    apt-get -y install apt-transport-https dotnet-runtime-6.0 && \
+    apt-get install dotnet-runtime-6.0 -y && \
     # install altV module
     wget --no-cache -q -O /opt/altv/modules/libcsharp-module.so https://cdn.altv.mp/coreclr-module/${BRANCH}/x64_linux/modules/libcsharp-module.so && \
     mkdir -p /usr/share/dotnet/host/fxr/ && \
     wget --no-cache -q -O /opt/altv/AltV.Net.Host.dll https://cdn.altv.mp/coreclr-module/${BRANCH}/x64_linux/AltV.Net.Host.dll && \
     # remove unused tools
-    apt-get purge -y wget gnupg && \
+    apt-get purge -y wget jq gnupg && \
```

Comment:
I think you can also remove `apt-transport-https` since it is only used as an installation-dependency to `packages-microsoft-prod.deb`. I am not that sure if it would work, but would be worth testing.

Reply:
Sorry, I meant cleaning up `apt-transport-https` - do not remove the complete package from the Dockerfile :)